sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -121,12 +121,14 @@ class APISpec extends Specification {
     /* Unignore any managed disk tests if a managed disk account is available to be tested. They are difficult to
      acquire so we do not run them in the nightly live run tests. */
     static def MANAGED_DISK_STORAGE = "MANAGED_DISK_STORAGE_"
+    static def VERSIONED_STORAGE = "VERSIONED_STORAGE_"
 
     protected static StorageSharedKeyCredential primaryCredential
     static StorageSharedKeyCredential alternateCredential
     static StorageSharedKeyCredential blobCredential
     static StorageSharedKeyCredential premiumCredential
     static StorageSharedKeyCredential managedDiskCredential
+    static StorageSharedKeyCredential versionedCredential
```

Comment:
did we create this new resource just cause versioning is still in stage? Would it eventually be available on any account?

Reply:
This is because versioning isn't on by default and can't be enabled from our data plane apis. It's similar in that way to a premium account, so we decided it'd be best to have a separate credential for it (though, as with all our credentials, it could be the same as the primary if the primary has this feature turned on).